### PR TITLE
Upgrade to Polkadot v1.0.0

### PR DESCRIPTION
**Pull request by lemunozm**
Upgrade to Polkadot v1.0.0

**Keep the branch**

### CODE REVIEW

File: mock-builder/tests/pallet.rs [hunk 1]
```diff
@@ -151,22 +151,12 @@ pub mod my_pallet {
...
-	type UncheckedExtrinsic = frame_system::mocking::MockUncheckedExtrinsic<Runtime>;
-	type Block = frame_system::mocking::MockBlock<Runtime>;
-
 	frame_support::construct_runtime!(
-		pub enum Runtime where
-			Block = Block,
-			NodeBlock = Block,
-			UncheckedExtrinsic = UncheckedExtrinsic,
-		{
+		pub struct Runtime {
```
Comment (lemunozm) [inline mock-builder/tests/pallet.rs:159]: I've found this change from Polkadot v1.0.0 pretty good 👍🏻

File: mock-builder/tests/pallet.rs [hunk 2]
```diff
@@ -177,17 +167,16 @@ mod mock {
 		type AccountData = ();
 		type AccountId = u64;
 		type BaseCallFilter = frame_support::traits::Everything;
+		type Block = frame_system::mocking::MockBlock<Runtime>;
```
Comment (lemunozm) [inline mock-builder/tests/pallet.rs:170]: The block information is now tracking by the `frame_system` 👌🏻

**Review by wischli — APPROVED** [review]
Thanks for working on this so quickly! I suppose v1.1.0 and v1.2.0 will be following?

**Comment by lemunozm** [comment]
Yeah, it is on the way!

**Pull request merged**